packages/core/src/InlineEditor/InlineEditor.tsx

```diff
@@ -46,6 +46,13 @@ export type HvInlineEditorProps<C extends React.ElementType = typeof HvInput> =
       ) => void;
       /** Called when the input value changes. */
       onChange?: (event: React.SyntheticEvent, value: string) => void;
+      /** Called when there's a keydown event on the input. */
+      onKeyDown?: (
+        event:
+          | React.KeyboardEvent<HTMLTextAreaElement | HTMLInputElement>
+          | React.MouseEvent,
```

Comment:
This `React.MouseEvent` on an `onKeyDown` is really strange... but it's coming from `HvInput` 🤷